src/contactForm.js

```diff
@@ -178,6 +214,22 @@ class RecrasContactForm {
         return this.hasFieldOfType('boeking.arrangement');
     }
 
+    isEmpty() {
+        let isEmpty = true;
+        let els = this.findElements('.recras-contactform input, .recras-contactform select, .recras-contactform textarea');
+        let formValues = [...els].map(el => el.value);
+        for (let val of formValues) {
+            if (val !== '') {
+                isEmpty = false;
```

Comment:
I think this function would improve a bit with an early return, or [`Array.prototype.some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)